WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,14 +1,7 @@
-Blocks are now drag-and-drop, so you can easily rearrange your content however you like. We like to move it, move it.
+* [*] Quick Start: The "Get to know the WordPress app" card has a fresh new look [#18688, #18747]
```

Comment:
The Jetpack notes are the same as WordPress, although it’s worth noting that the “Get to know the WordPress app” card reads “Get to know the Jetpack app” there.

![IMG_8D9D7EB4BDF1-1](https://user-images.githubusercontent.com/1218433/170918653-7841cad6-0c31-4bb9-87ca-a12bdcb96242.jpeg)



I don’t think this is something worth tweaking in the raw release notes, but I’ll make sure to let our copy editor know.